modules/administration/pages/hostname-rename.adoc

```diff
@@ -0,0 +1,17 @@
+[[hostname-rename]]
+= Rename Hostname
+
+With the [command]``spacewalk-hostname-rename`` script it is possible to rename the hostname of the {productname} Server.
+The [command]``spacewalk-hostname-rename`` script is part of the [package]``spacewalk-utils`` package.
+
+The only mandatory parameter for the script is the newly configured IP address of the {productname} Server.
+It is the best to proceed as follows: 
+
+. Adjust the hostname in advance.
+It is important to configure it locally, and for DNS resolution (including reverse resolution).
+. Load the new configuration.
```

Comment:
This is not about new configuration, but about the server using the new hostname. Otherwise the script complains.

Reply:
ok, I changed it for clarity.